packages/site-demo/content/product/components/thumbnail/index.mdx

```diff
@@ -34,9 +34,26 @@ Thumbnails come in two variants: `squared` and `rounded`
 
 <DemoBlock orientation="horizontal" demo="combined" />
 
-## Loading and error states
+## Loading skeleton
```

Comment:
"Loading ~skeleton~state"

Skeleton is the component that make possible l[oading states](https://design.lumapps.com/product/patterns/loading-state/)